doc/sphinx/make-your-own-compositor.md

```diff
@@ -0,0 +1,248 @@
+---
```

Comment:
This file should be renamed to `tutorial.md` in my opinion

Reply:
Having descriptive names never hurts :)

Reply:
Or `write-your-first-wayland-compositor.md`